one-pager-maker/src/api/documentApi.ts

```diff
@@ -6,7 +6,23 @@ import { Document } from "../entity/documentType.ts";
 
 const queryKeys = {
   documentId: (uid: string, documentId: string) =>
-    `document-${uid}-${documentId}`
+    `document-${uid}-${documentId}`,
+  documentsUnderParent: (uid: string, parentId?: string) =>
```

Comment:
What does "documents Under Parent" mean?

Reply:
@binnmti 
`documentsUnderParent` refers to a hierarchical structure in document management where documents can be nested or organized under other documents.

Reply:
I'm not sure if it's the best name, but it could just be that I don't understand it well, and it's something I can change later, so it's OK!